framework/include/media/media_player.h

```diff
@@ -0,0 +1,76 @@
+/****************************************************************************
+ *
+ * Copyright 2017 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __MEDIA_PLAYER_H__
+#define __MEDIA_PLAYER_H__
+
+#include <sys/types.h>
+#include <stdbool.h>
+#include <media/media_types.h>
+#include <sched.h>
+#include <tinyalsa/tinyalsa.h>
+
+/***
+ * Enumerations
+ */
+typedef enum {
```

Comment:
enum play_state_e {
}
typedef play_state_e play_state_t is correct rule..

Reply:
Changed to correct to the coding rule.